Enable wallace after configuration is complete.
Wallace is disabled when installed.

!! Temporarily disable Wallace because of bug #1469 !!
See: https://issues.kolab.org/show_bug.cgi?id=1469

--- a/pykolab/setup/setup_mta.py
+++ b/pykolab/setup/setup_mta.py
@@ -319,7 +319,15 @@
             myaugeas.set(setting,'1')
             myaugeas.save()
         myaugeas.close()
-        
+
+    if os.path.isfile('/etc/default/wallace'):
+        myaugeas = Augeas()
+        setting = os.path.join('/files/etc/default/wallace','START')
+        if not myaugeas.get(setting) == 'no':
+            myaugeas.set(setting,'no')
+            myaugeas.save()
+        myaugeas.close()
+
     if os.path.isfile('/bin/systemctl'):
         subprocess.call(['systemctl', 'restart', 'postfix.service'])
         subprocess.call(['systemctl', 'restart', 'amavisd.service'])
